src/components/molecules/idCopyButton.tsx

```diff
@@ -1,47 +1,18 @@
 import React from "react";
 
-import { debounce } from "lodash";
 import { useTranslation } from "react-i18next";
 
-import { useToastStore } from "@src/store";
-import { cn, copyToClipboard } from "@src/utilities";
-
-import { IconButton } from "@components/atoms";
-
-import { CopyIcon } from "@assets/image/icons";
+import { CopyButton } from "./copyButton";
```

Comment:
import to component should start from `@`

Reply:
done